mcp300x_block.py

```diff
@@ -91,11 +92,13 @@ def _read_from_channel(self, channel):
         # start bit
         bytes_to_send.append(1)
         # channel number in most significant nibble
-        # Note: the 8 sets the mode to "single" instead of "differential"
```

Comment:
I'm thinking some comments could be useful here.  I don't really know what's going on here but at least this comment helps me think I understand a little bit.

And what's up with total_channels? The number doesn't really matter? It's more just checking if it's a multiple of four and then does some different bit setting?